common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -239,7 +245,10 @@ def get_html(self):
             'element_class': self.location.category,
             'ajax_url': self.system.ajax_url,
             'num_inputs': self.num_inputs,
-            'submitted': self.submitted
+            'submitted': self.submitted,
+            'instructions_default': WordCloudFields.instructions.default,
+            'instructions': self.instructions,
+            'display_name': self.display_name
```

Comment:
Is that how it works with other components? Do we hide the default name, or hide it if it's unchanged?